chain/network/src/ibf.rs

```diff
@@ -0,0 +1,249 @@
+use std::cmp::{max, min};
+use std::hash::Hasher;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use std::collections::hash_map::DefaultHasher;
+use tracing::error;
+
+const NUM_HASHES: usize = 3;
```

Comment:
Please add a comment on what this constant is used for and why it is set to 3

Reply:
Added exmplanation.